File: samples/apps-in-meeting/nodejs/Content Bubble Bot  Nodejs/models/taskmoduleuiconstants.js
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

const { UISettings } = require('./uisettings');
const { TaskModuleIds } = require('./taskmoduleids');

const TaskModuleUIConstants = {
    contentBubble: new UISettings(510, 450, 'Content Bubble', TaskModuleIds.contentBubble, 'Content Bubble'),
    contentBubbleTwo: new UISettings(510, 450, 'Content Bubble Two', TaskModuleIds.contentBubbleTwo, 'Content Bubble'),
    contentBubbleThree: new UISettings(510, 450, 'Content Bubble Three', TaskModuleIds.contentBubbleThree, 'Content Bubble'),
    AdaptiveCard: new UISettings(400, 200, 'Adaptive Card: Inputs', TaskModuleIds.AdaptiveCard, 'Adaptive Card')
};

module.exports.TaskModuleUIConstants = TaskModuleUIConstants;

File: samples/bot-task-module/python/models/task_module_ui_constants.py
# Copyright (c) Microsoft Corp. All rights reserved.
# Licensed under the MIT License.

from .ui_settings import UISettings
from .task_module_ids import TaskModuleIds


class TaskModuleUIConstants:
    """
    A class to hold constants for Task Module UI settings.
    """
    
    # UI settings for YouTube video task module
    YOUTUBE = UISettings(1000, 700, "YouTube Video", TaskModuleIds.YOUTUBE, "YouTube")
    
    # UI settings for Custom Form task module
    CUSTOM_FORM = UISettings(
        510, 450, "Custom Form", TaskModuleIds.CUSTOM_FORM, "Custom Form"
    )
    
    # UI settings for Adaptive Card task module
    ADAPTIVE_CARD = UISettings(
        400, 200, "Adaptive Card: Inputs", TaskModuleIds.ADAPTIVE_CARD, "Adaptive Card",
    )
